WPMedia/arc-themes-components
# Tmedia 673 media item

# Media item 

- [TMEDIA-673](https://arcpublishing.atlassian.net/jira/software/c/projects/TMEDIA/boards/875?modal=detail&selectedIssue=TMEDIA-673)

## Description

_Information about what you changed for this PR_

## Acceptance Criteria

_copy from ticket_

## Test Steps

1. Checkout branch - `git checkout {branch-name}`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Check out the Media Item storybook
5. Notice the src passed in in the new Media Item
6. The height and width can be determined like the Image component 
7. If falsy src then don't render image

## Questions: 

- _Are we using the powa player for videos here?_ Yes, but in the video component. the video implementation is deferred for this ticket 
- _For determining a video or an image, I was thinking of a prop like `mediaType` to either render an img or a video within this component._ This is deferred for now but this is something to think about. 
- Just to clarify, we won't be nesting the Image component within this MediaItem component like 
```jsx
const Feature = () => (
<MediaItem>
<Image>
</MediaItem>
)
```
_Instead, we want to duplicate the props and options necessary for showing an image within the MediaItem, right?_ 

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
